client/src/components/ComputerSysContainer.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import ComputerSysDescription from './ComputerSysDescription';
+import ComputerSysMoreDetails from './ComputerSysMoreDetails';
+import {Tabs, Tab } from 'react-bootstrap';
+import '../css/style.css';
+
+/**
+ * Component to show the selected computer system information.
+ */
+class ComputerSysContainer extends Component {
```

Comment:
Nitpick: This could be stateless.

Reply:
Yes, I believe you are correct. Since this is just a presentation component it doesn't make sense not to be a stateless one.